test/prism/PrismLogicTest.scala

```diff
@@ -265,4 +265,32 @@ class PrismLogicTest extends FreeSpec with Matchers {
       percentiles.p90 shouldEqual Some(14)
     }
   }
+
+  "instancesCountPerSSA" - {
+    "returns " in {
```

Comment:
This won't be very helpful in a test output log when it fails. How about "returns the expected result", or perhaps more helpfully, "correctly generates and associates the instance count with each SSA"?